expressEngine/lib/successHandler/successHandler.js

```diff
@@ -1,6 +1,3 @@
-export default function successHandler(message, data = null) {
-  return ({
-    data,
-    message,
-  });
-}
+import success from '../../cms/common/success';
+
+export default (message = success.defaultSuccessMsg, data = null) => ({ data, message });
```

Comment:
@chiragSuccessive Instead of doing that I'll prefer if we can create a responseHelper which can have two methods like success and error. It'll handle rest of the things there.